test/views/declaration/TransportContainerAddFirstViewSpec.scala

```diff
@@ -50,6 +51,27 @@ class TransportContainerAddFirstViewSpec extends UnitViewSpec with ExportsTestDa
       view.getElementById("section-header") must containMessage("declaration.section.6")
     }
 
+    "display radio button with Yes option" in {
+      view.getElementById("code_yes").attr("value") mustBe YesNoAnswers.yes
+      view.getElementsByAttributeValue("for", "code_yes") must containMessageForElements("declaration.transportInformation.containers.yes")
+    }
+
+    "display radio button with No option" in {
+      view.getElementById("code_no").attr("value") mustBe YesNoAnswers.no
+      view.getElementsByAttributeValue("for", "code_no") must containMessageForElements("declaration.transportInformation.containers.no")
+    }
+
+    "display inset text" in {
+      val inset = view.getElementsByClass("govuk-inset-text")
+      val expected = Seq(
+        messages("declaration.transportInformation.containers.inset.title"),
+        messages("declaration.transportInformation.containers.inset.bullet1"),
+        messages("declaration.transportInformation.containers.inset.bullet2"),
+        messages("declaration.transportInformation.containers.inset.bullet3")
+      ).mkString(" ")
+      inset.get(0) must containText(expected)
```

Comment:
Nit: Usually I've seen such things to be tested with: `inset.get(0) must include(messages("declaration..."))` for each message you need to test. Not sure if there is much difference though.